app/serializers/api/v2/certificates/certificates_serializer_list.rb

```diff
@@ -0,0 +1,23 @@
+module Api
+  module V2
+    module Certificates
+      class CertificatesSerializerList
```

Comment:
I added a new serializer in order to avoid conditional attributes (it require passing in param attributes, which makes things hard to read: https://github.com/jsonapi-serializer/jsonapi-serializer#conditional-attributes)